chart/openfaas/templates/queueworker-dep.yaml

```diff
@@ -28,6 +34,18 @@ spec:
         - name: ack_wait    # Max duration of any async task / request
           value: {{ .Values.queueWorker.ackWait }}
         {{- end }}
+        - name: secret_mount_path
```

Comment:
The above `{{- if .Values.functionNamespace }}` will remove the `env` key and the yaml will be invalid. Also the `secret_mount_path` should be included in the `{{- if .Values.basic_auth }}`

Reply:
I have updated it.